.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "\U0001F41B Bug report"
+about: Report that something isn't working as expected.
+title: ''
+labels: type:bug
+assignees: ''
+
+---
+
+## Type of report
```

Comment:
We can remove "Type of report" section for these templates as it will be redundant to the (auto assigned) label.